static/docs/commands-reference/install.md

```diff
@@ -35,12 +35,23 @@ is a reminder to run `dvc commit`.
 The installed Git hook automates reminding the user to run either `dvc repro` or
 `dvc commit`.
 
+**Push** Sharing data across environments and making changes to remote storage
+is done by this command. This command searches for changed .dvc files and stages
```

Comment:
this command -> `dvc push`

Reply:
.dvc files -> DVC-files

Reply:
it's not looking for changed files, it takes all DVC-files from the current workspace

Reply:
ok,corrected.